webui/src/actions/actions.jsx

```diff
@@ -1,287 +1,260 @@
 }
 
-function newBuild(buildID) {
+export function newBuild(buildID) {
     return {
         type: actionTypes.NEW_BUILD,
         buildID: buildID
     }
 }
 
-function newBuildError(errorMessage) {
+export function newBuildError(errorMessage) {
     return {
         type: actionTypes.NEW_BUILD_ERROR,
         errorMessage: errorMessage
     }
 }
 
-function fileUploadSuccess(data) {
+export function fileUploadSuccess(data) {
     return {
         type: actionTypes.FILE_UPLOAD_SUCCESS,
         data: data
     }
 }
 
-function fileUploadError(errorMessage) {
+export function fileUploadError(errorMessage) {
     return {
         type: actionTypes.FILE_UPLOAD_ERROR,
         errorMessage: errorMessage
     }
 }
 
-function inspectVolumeStart() {
+export function inspectVolumeStart() {
     return {
         type: actionTypes.INSPECT_VOLUME_START
     }
 }
 
-function inspectVolumeSuccess(data) {
+export function inspectVolumeSuccess(data) {
     return {
         type: actionTypes.INSPECT_VOLUME_SUCCESS,
         data: data
     }
 }
 
-function inspectVolumeEmpty() {
+export function inspectVolumeEmpty() {
     return {
         type: actionTypes.INSPECT_VOLUME_EMPTY,
         data: []
     }
 }
 
-function inspectVolumeError(errorMessage) {
+export function inspectVolumeError(errorMessage) {
     return {
         type: actionTypes.INSPECT_VOLUME_ERROR,
         errorMessage: errorMessage
     }
 }
 
-function consoleOutputFetchStart() {
+export function consoleOutputFetchStart() {
     return {
         type: actionTypes.CONSOLE_OUTPUT_FETCH_START
     }
 }
 
-function consoleOutputFetchSuccess(data) {
+export function consoleOutputFetchSuccess(data) {
     return {
         type: actionTypes.CONSOLE_OUTPUT_FETCH_SUCCESS,
         data: data
     }
 }
 
-function consoleOutputFetchEmpty() {
+export function consoleOutputFetchEmpty() {
     return {
         type: actionTypes.CONSOLE_OUTPUT_FETCH_EMPTY,
         data: []
     }
 }
 
-function consoleOutputFetchError(errorMessage) {
+export function consoleOutputFetchError(errorMessage) {
     return {
         type: actionTypes.CONSOLE_OUTPUT_FETCH_ERROR,
         errorMessage: errorMessage
     }
 }
 
-function ioAddStart() {
+export function ioAddStart() {
     return {
         type: actionTypes.IO_ADD_START
     }
 }
 
-function ioAddSuccess(data) {
+export function ioAddSuccess(data) {
     return {
         type: actionTypes.IO_ADD_SUCCESS,
         data: data
     }
 }
 
-function ioAddError(errorMessage) {
+export function ioAddError(errorMessage) {
     return {
         type: actionTypes.IO_ADD_ERROR,
         errorMessage: errorMessage
     }
 }
 
-function ioRemoveStart() {
+export function ioRemoveStart() {
     return {
         type: actionTypes.IO_REMOVE_START
     }
 }
 
-function ioRemoveSuccess(data) {
+export function ioRemoveSuccess(data) {
     return {
         type: actionTypes.IO_REMOVE_SUCCESS,
         data: data
     }
 }
 
-function ioRemoveError(errorMessage) {
```

Comment:
We still have IO_REMOVE_ERROR in actionTypes. How do we report about errors for IO removal now?